src/Workspaces/CoreTest/UtilityTest/FormattingRangeHelperTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests.UtilityTest
+{
+    public class FormattingRangeHelperTests
+    {
+        [Fact]
+        public void TestAreTwoTokensOnSameLineTrue()
```

Comment:
💡 Can you include `[WorkItem]` attributes so these tests link back to the original bug? At minimum, you want to include this attribute on all tests which fail without your change above. However, it would be fine to include the attribute for all tests you added here, even if some of them worked before and after the change.

Reply:
Thanks, I have added the WorkItem attribution to the tests.